### ISSUE TITLE
How to solve Xaml parsing error for media.plugin in windows phone 8.1 PCL project?

### ISSUE THREAD

**Issue by jamesmontemagno** [jamesmontemagno/MediaPlugin#16]
_From @rylegale on April 5, 2016 9:22_

Please take a moment to fill out the following (change to preview to check or place x in []):

This is a Bug
- [  ] Bug
- [ ] Feature Request

Which plugin does this impact: Media
- [ ] Battery
- [ ] Connectivity
- [ ] Contacts
- [ ] DeviceInfo
- [ ] ExternalMaps
- [ ] Geolocator
- [  ] Media
- [ ] Permissions
- [ ] Settings
- [ ] Text To Speech
- [ ] Vibrate
- Other:

Version Number of Plugin: 2.3.0
Device Tested On: 
Simulator Tested On: Windows Phone 8.1

### Actual Behavior

XAML parsing failed.

### Steps to reproduce the Behavior

I've created a PCL project and the plugin is working so good in Android and iOS but not in Windows Phone 8.1 RT, I've added the lines in App.xaml.cs but when I try to take picture, it throws an error XAML parsing failed in TakePhotoAsync method. I've also tried to download and manually added the dll file but again the same error.

_Copied from original issue: jamesmontemagno/Xamarin.Plugins#269_

**Comment by jamesmontemagno** [comment]
_From @rylegale on April 5, 2016 10:23_

```
at Windows.UI.Xaml.Application.LoadComponent(Object component, Uri resourceLocator, ComponentResourceLocation componentResourceLocation)
   at DMX.Helper.CameraCaptureUI.InitializeComponent()
   at DMX.Helper.CameraCaptureUI..ctor()
   at Plugin.Media.MediaImplementation.<TakePhotoAsync>d__16.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at SwachhParyatanApp.Complaint.<>c__DisplayClass1_0.<<-ctor>b__0>d.MoveNext()
```

**Comment by jamesmontemagno** [comment]
_From @avorobjovs on May 30, 2016 20:17_

Hello!

I have installed the Xam.Plugin.Media (version 2.3.0) on my Xamarin.Forms (version 2.1.0.6529, PCL) project and have the same problem. I have tested it on 2 WindowsPhone devices and get such exceptions on both of them while I call the TakePhotoAsync() method. At the same time the PickPhotoAsync() method works good.

Do you have a plan to fix this bug?

This feature (take a photo) is a critical for our application.

Best Regards,
Aleksandrs

**Comment by jamesmontemagno** [comment]
_From @FBerco on June 8, 2016 21:1_

Same problem here @avorobjovs !

Any solution? @jamesmontemagno

**Comment by jamesmontemagno** [comment]
_From @rylegale on June 9, 2016 5:0_

@FBerco I found the solution, you can download the sourcecode from github and then open it in visual studio and manually compile it and make sure to copy the two cameracapture xaml file manually to your project and it'll e working file

**Comment by jamesmontemagno** [comment]
_From @FBerco on June 9, 2016 16:9_

@rylegale CameraCaptureUI.xaml and CameraCaptureUiPage.xaml?? To where directory do I paste it?

**Comment by jamesmontemagno** [comment]
Please test: 2.5.0-beta5

**Comment by jamesmontemagno** [comment]
Fixed officially in 2.5.1-beta33! Tested and shipped!
